install.js

```diff
@@ -0,0 +1,89 @@
+var fs = require("fs");
+var os = require("os");
+var URL = require("url");
+var pkg = require("./package");
+
+function httpRequest(url, method, response) {
```

Comment:
Use a slightly-higher-level lib like [`simple-get`](https://www.npmjs.com/package/simple-get) for this. Unfortunately we'll still disrepect other important parts of the HTTP ecosystem, e.g. caching & proxy support.